Support Vue JSX (#4897)
withastro/astro: packages/integrations/vue/README.md

  })],
}
```

### jsx

You can use Vue JSX by setting `jsx: true`.

> 
> ```js
> // astro.config.mjs

Reviewer: ```suggestion
__`astro.config.mjs`__

```js
```

Reviewer: Just so it shows up as the code tab name when we pull the readme into docs!

import { defineConfig } from 'astro/config';
import vue from '@astrojs/vue';

export default defineConfig({
  integrations: [
    vue({ jsx: true })
  ],
});
```

> This will enable rendering both Vue and Vue JSX components. To customize the Vue JSX compiler, options can be passed instead of a boolean. See the `@vitejs/plugin-vue-jsx` [docs](https://github.com/vitejs/vite/tree/main/packages/plugin-vue-jsx) for more details.

Reviewer: ```suggestion
This will enable rendering for both Vue and Vue JSX components. To customize the Vue JSX compiler, pass an options object instead of a boolean. See the `@vitejs/plugin-vue-jsx` [docs](https://github.com/vitejs/vite/tree/main/packages/plugin-vue-jsx) for more details.
```
